package.json

```diff
@@ -24,7 +24,6 @@
 		"check": "tsc --noEmit",
 		"pretty": "prettier --write '**/*.{js,scss}'",
 		"install-git-hooks": "rm -f ./.git/hooks/* && cp ./.githooks/* ./.git/hooks && chmod +x ./.git/hooks/*",
-		"postinstall": "npm run build-prod",
```

Comment:
This might be necessary for heroku somewhere, but if so I'd like to do it without the postinstall hook (or at least make it conditional on some heroku only env var)

Reply:
Not necessary! https://help.heroku.com/P5IMU3MP/heroku-node-js-build-script-change-faq